governance.md

```diff
@@ -68,26 +65,27 @@ The foundations of Project governance are:
 -   Active Contribution
 -   Institutional Neutrality
 
-Traditionally, Project leadership was provided by a BDFL (Fernando Perez) and
-subset of Contributors, called Core Developers, whose active and consistent
-contributions have been recognized by their receiving “commit rights” to the
-Project GitHub repositories. In general all Project decisions are made through
-consensus among the Core Developers with input from the Community. The BDFL
-can, but rarely chooses to, override the Core Developers and make a final
-decision on a matter.
-
-While this approach has served us well, as the Project grows and faces more
-legal and financial decisions and interacts with other institutions, we see a
-need for a more formal governance model. Moving forward The Project leadership
-will consist of a BDFL and Steering Council. We view this governance model as
-the formalization of what we are already doing, rather than a change in
-direction.
+Project leadership was provided by the original author, John
+D. Hunter.  Since his passing in 2012, leadership has been provided by
+a subset of Contributors, called Core Developers, whose active and
+consistent contributions have been recognized by their receiving
+“commit rights” to the Project repositories. In general all Project
+decisions are made through consensus among the Core Developers with
+input from the Community.
+
+While this approach has served us well, as the Project grows and faces
+more legal and financial decisions and interacts with other
+institutions, we see a need for a more formal governance model. Moving
+forward The Project leadership will consist of a Steering Council with
+the power to appoint final decisions to an individual. We view this
```

Comment:
Yeah, I'm not sure how "with the power to appoint final decisions to an individual" is better than "a BDFL", as was in the original text from jupyter.